File: eval/baselines/dashboard/jetpack-compose/DashboardScreen.kt
// BASELINE — intentionally-naive "no-skill" reference for the eval harness (do NOT
// copy). What an assistant typically emits WITHOUT the Mobile UI/UX Pro Max skill:
// hardcoded colors, off-grid spacing, sub-legible captions, a fixed 2-column grid
// (no window size classes), and only the happy path with none of the required UI
// states. Graded against examples/dashboard/jetpack-compose/.
package baseline.dashboard

import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.lazy.grid.GridCells
import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
import androidx.compose.foundation.lazy.grid.items
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp

@Composable
fun DashboardScreen() {
    val tiles = listOf("Revenue", "Orders", "Visitors", "Refunds")
    Column(modifier = Modifier.padding(18.dp)) {
        Text("Dashboard", color = Color(0xFF111827), fontSize = 22.sp)
        LazyVerticalGrid(columns = GridCells.Fixed(2)) {
            items(tiles) { t ->
                Column(modifier = Modifier.padding(10.dp).background(Color(0xFFFFFFFF))) {
                    Text(t, color = Color(0xFF111827))
                    Text("+12%", color = Color(0xFF16A34A), fontSize = 10.sp)
                }
            }
        }
    }
}

File: eval/baselines/dashboard/flutter/dashboard_screen.dart
// BASELINE — intentionally-naive "no-skill" reference for the eval harness (do NOT
// copy). What an assistant typically emits WITHOUT the Mobile UI/UX Pro Max skill:
// hardcoded colors, off-grid spacing, physical left/right insets, a fixed-height
// metric row, sub-legible captions, a fixed 2-column grid (no size classes), and
// only the happy path with none of the required UI states. Graded against
// examples/dashboard/flutter/.
import 'package:flutter/material.dart';

class DashboardScreen extends StatelessWidget {
  const DashboardScreen({super.key});

  @override
  Widget build(BuildContext context) {
    final tiles = ['Revenue', 'Orders', 'Visitors', 'Refunds'];
    return Scaffold(
      backgroundColor: const Color(0xFFF9FAFB),
      appBar: AppBar(title: const Text('Dashboard')),
      body: Padding(
        padding: const EdgeInsets.only(left: 18, right: 18, top: 10),
        child: GridView.count(
          crossAxisCount: 2,
          mainAxisSpacing: 15,
          crossAxisSpacing: 15,
          children: [
            for (final t in tiles)
              Container(
                padding: const EdgeInsets.all(10),
                color: const Color(0xFFFFFFFF),
                child: Column(
                  crossAxisAlignment: CrossAxisAlignment.start,
                  children: [
                    Container(height: 22, child: Text(t, style: const TextStyle(color: Color(0xFF111827)))),
                    const Text('+12%', style: TextStyle(fontSize: 10, color: Color(0xFF16A34A))),
                  ],
                ),
              ),
          ],
        ),
      ),
    );
  }
}
